engines/engine.cpp

```diff
@@ -507,21 +521,30 @@ void Engine::checkCD() {
 			"the data files to your hard disk instead.\n"
 			"See the Documentation (CD audio) for details."), _("OK"));
 		dialog.runModal();
-	} else {
-#endif // defined(WIN32) && !defined(__SYMBIAN32__)
-		// If we reached here, the game has audio tracks,
-		// it's not ran from the CD and the tracks have not
-		// been ripped.
-		GUI::MessageDialog dialog(
-			_("This game has audio tracks in its disk. These\n"
-			"tracks need to be ripped from the disk using\n"
```

Comment:
I wouldn't write "in its disk". Rather "on the disk" or "on the CD". However, I am not a native speaker of English, so be careful with my advice here...

Reply:
I put "on its CD" because it sounds better to me. Also replaced the "disk" in the next sentence with "CD". These changes are in commit:
https://github.com/scummvm/scummvm/pull/3018/commits/3ec3820ddbf70905a2aaba6d7c0374d1f77f796a